basics/5 terminal/index.md

```diff
@@ -1,27 +1,147 @@
-# Getting started with Git and the terminal
```

Comment:
In het algemeen: ze hebben een zeer uitgebreid college gekeken vooraf, dus deze pagina moet van bovenaan duidelijk maken dat studenten het zelf moeten uitproberen. Misschien noemen we het een tutorial (niet in de kop hoor, maar in de introtekst).

Iets van:
"You have seen how **git** works in lecture, and now it's your turn to put your hands on the keyboard! Make sure that you follow the steps of the tutorial on your own computer, and get to know how it all works in practice."